pkg/apis/core/validation/validation_test.go

```diff
@@ -10059,6 +10059,7 @@ func TestValidatePodEphemeralContainersUpdate(t *testing.T) {
 
 func TestValidateServiceCreate(t *testing.T) {
 	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.ServiceTopology, true)()
+	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.ServiceInternalTrafficPolicy, true)()
```

Comment:
Wouldn't this validation happen even if the feature gate wasn't enabled?

Reply:
you're right. removed this line